src/bumerang/borrowsbyrecenthandler.py

```diff
@@ -0,0 +1,30 @@
+from bumerang.bumerangrequesthandler import BumerangRequestHandler
+from bumerang.error import BumerangError
+from json import dumps
+
+class BorrowsByRecentHandler(BumerangRequestHandler):
+    """ This class handles fetching multiple bumerang requests depending
+        on which are most recently created.
+    """
+
+    def get(self, num_requests=10):
+        """Obtain a list of requests."""
+
+        try:
+            num = int(num_requests)
+            requests = self.borrow_repo.find_requests_by_recent(num)
+            if requests:
+                nodes = [req.to_node() for req in requests]
+                json_string = dumps(nodes)
+                self.write(json_string)
```

Comment:
In some common apis results are returned as `'results': ['result': {node}, 'result': ...]`